Source/WebCore/platform/graphics/FontDescription.h

```diff
@@ -86,6 +86,33 @@ class FontDescription {
     FontVariantEastAsianWidth variantEastAsianWidth() const { return static_cast<FontVariantEastAsianWidth>(m_variantEastAsianWidth); }
     FontVariantEastAsianRuby variantEastAsianRuby() const { return static_cast<FontVariantEastAsianRuby>(m_variantEastAsianRuby); }
     FontVariantEmoji variantEmoji() const { return static_cast<FontVariantEmoji>(m_variantEmoji); }
+    FontVariantEastAsianValues variantEastAsian() const
+    {
+        return {
+            variantEastAsianVariant(),
+            variantEastAsianWidth(),
+            variantEastAsianRuby(),
+        };
+    }
+    FontVariantNumericValues variantNumeric() const
+    {
+        return {
+            variantNumericFigure(),
+            variantNumericSpacing(),
+            variantNumericFraction(),
+            variantNumericOrdinal(),
+            variantNumericSlashedZero(),
+        };
+    }
+    FontVariantLigaturesValues variantLigatures() const
+    {
+        return {
+            variantCommonLigatures(),
+            variantDiscretionaryLigatures(),
+            variantHistoricalLigatures(),
+            variantContextualAlternates(),
+        };
+    }
```

Comment:
I would put these vertically formatted function bodies outside the class definition.